server/src/main/java/org/opensearch/search/fetch/subphase/highlight/AbstractHighlighterBuilder.java

```diff
@@ -189,7 +189,7 @@ protected AbstractHighlighterBuilder(StreamInput in) throws IOException {
 
         requireFieldMatch(in.readOptionalBoolean());
 
-        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
+        if (in.getVersion().onOrAfter(Version.V_2_2_0)) {
```

Comment:
👍 @hauck-jvsh we would need to bring this change back to `main`

Reply:
Yes, how is the way to do this?

Reply:
Sadly, manual PR (but this should be the last one)